easybuild/easyconfigs/c/cuDNN/cuDNN-7.0.2-CUDA-9.0.176.eb

```diff
@@ -0,0 +1,34 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+easyblock = 'Tarball'
+
+name = 'cuDNN'
+version = '7.0.2'
+cuda_version = '9.0.176'
+
+cuda_version_major_minor = '.'.join(cuda_version.split('.')[:2])
+
+versionsuffix = '-CUDA-%s' % cuda_version
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for
+    deep neural networks."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Nvidia developer registration required.
+# Download link: https://developer.nvidia.com/rdp/cudnn-download
+sources = ['%%(namelower)s-%s-linux-x64-v%%(version)s.tgz' % cuda_version_major_minor]
```

Comment:
@smoors Should use `%%(version_major)s`, since the tarball name is `cudnn-9.0-linux-x64-v7.tgz`?

Reply:
yeah, I was thinking it is better to rename the tarball to the full version number, but you are probably right :)

Reply:
Well, good point, but then you should use EasyBuild's renaming support, see the use of `download_filename` at http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources

Reply:
works as expected with the change in 94a68d5, let me know if you want to rename it to include the full version or not...

Reply:
I'll try to make the rename change